classes/PluginInstall.class.php

```diff
@@ -68,7 +68,7 @@ public function rename_menu() {
 
 	// Get all installed ClassicPress plugin
 	// This function is different from the one in PluginUpdate class
-	// and considers a plugin from the dir not only if it has UpdateURI
+	// and considers a plugin from the dir not only if it has RequiresCP
 	// but also if it have RequiresCP.
```

Comment:
// but also if it has RequiresCP.

Reply:
It's line 72 that needs changing; line 71 should be as it was with `UpdateURI`

Reply:
Should be fixed.